.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow is used to create a release
+
+name: Create Release
+
+on:
+# Triggers the workflow on a tag push on master
+  push:
+    branches:
+      - master
+
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
```

Comment:
What is the purpose of the trailing `*` here? This doesn't make sense in a normal regex context in conjunction with the `+` preceding it.